src/device/device-management.ts

```diff
@@ -86,7 +87,16 @@ export async function createMicrophoneAndCameraTracks(
 export async function createCameraTrack(constraints?: VideoConstraints): Promise<LocalCameraTrack> {
   let stream: MediaStream;
   try {
-    // TODO: change it
+    if (
+      constraints?.encoderConfig &&
+      typeof constraints.encoderConfig === 'string' &&
+      encoderConfigMap.has(constraints.encoderConfig)
+    ) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      // eslint-disable-next-line no-param-reassign
+      constraints.encoderConfig = encoderConfigMap.get(constraints.encoderConfig);
+    }
     stream = await media.getUserMedia({ video: {} });
```

Comment:
@arun3528 , do we need to convert constraints into video object here?

Reply:
yes we will have to